README.md

```diff
@@ -1,44 +1,44 @@
 # Sonar Cryptography Plugin
 
+[![License](https://img.shields.io/github/license/IBM/sonar-cryptography.svg?)](https://opensource.org/licenses/Apache-2.0) <!--- long-description-skip-begin -->
+[![Current Release](https://img.shields.io/github/release/IBM/sonar-cryptography.svg?logo=IBM)](https://github.com/IBM/sonar-cryptography/releases)
+
+
 This repository contains a SonarQube Plugin that detects cryptographic assets 
-in source code and generates CBOM.
+in source code and generates [CBOM](https://cyclonedx.org/capabilities/cbom/).
 
-## Plugin version compatibility
+## Version compatibility
 
 | Plugin Version | SonarQube Version    |
-| -------------- | -------------------- |
+|----------------|----------------------|
 | 1.x.x and up   | SonarQube 9.8 and up |
 
-## Latest supported languages and libraries
+## Supported languages and libraries
 
-| Language | Cryptographic Library | Coverage | 
-| -------- | --------------------- | -------- |
-| Java     | JCA                   | 100%     |
-|          | BouncyCastle          | 80%      |
-| Pyrhon   | pycrypto              | 100%     |
+| Language | Cryptographic Library                                                                         | Coverage | 
+|----------|-----------------------------------------------------------------------------------------------|----------|
+| Java     | [JCA](https://docs.oracle.com/javase/8/docs/technotes/guides/security/crypto/CryptoSpec.html) | 100%     |
+|          | [BouncyCastle](https://github.com/bcgit/bc-java)                                              | 80%      |
+| Python   | pycrypto                                                                                      | 100%     |
```

Comment:
This is not pycrypto but [`pyca/cryptography`](https://cryptography.io/en/latest/)